scripts/index-g6-in-elasticsearch.py

```diff
@@ -87,5 +87,6 @@ def main():
     else:
         print(__doc__)
 
+
```

Comment:
We should really clear out scripts from Search API and API, either move them to scripts repo or delete (this one is a good candidate for delete!)

But that's for another time I guess.

Reply:
Sounds like  a good tech debt ticket. :trollface: